src/EnergyPlus/ChillerAbsorption.cc

```diff
@@ -1222,7 +1222,7 @@ void BLASTAbsorberSpecs::sizeChiller(EnergyPlusData &state)
                             BaseSizer::reportSizerOutput(state,
                                                          moduleObjectType,
                                                          this->Name,
-                                                         "Iniital Design Size Design Generator Fluid Flow Rate [m3/s]",
+                                                         "Initial Design Size Design Generator Fluid Flow Rate [m3/s]",
```

Comment:
Seems like this affects output?

Reply:
Yeah probably worth mentioning at least.  I can try to collect the changes here that "affect" output and add it to the new 25.1 output changes file.